### PR TITLE
[Mostly Not Modular]Additional language: Dawn of the polyglots edition

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

It's that additional languages PR but less shitty.
Instead of getting a language by joining the game as a certain race, you can choose your secondary language in the character setup. Hooray.

## Why It's Good For The Game

i want to speak beachbum language and be a cool surfer dude

## Changelog
:cl:
add: Additional language option in the character setup. Some races are still locked behind species however, as in, you cannot choose a secondary language that is race exclusive and doesn't pertain to your species.
add: A bunch of new languages. Notable ones include: Dunmeri, the language of the ashlander elves of lavaland. Neo-Kanji, an asian rooted language that is considered the official language of the spider clan. Technorussian, a language based on eastern european dialects (mostly russian of course). Sol Common, the bog standard human language.
tweak: Space ninjas now know how to speak Neo-Kanji by default.
fix: Humanoids and Dunmer are back.
/:cl:

### CODE REVIEW

File: code/modules/surgery/organs/tongue.dm [hunk 1]
```diff
@@ -25,7 +25,15 @@
...
-		/datum/language/machine //Skyrat change - adds machine language
+		/datum/language/machine, //Skyrat change - adds machine language
+		/datum/language/calcic, //Skyrat change - plasmaman language
+		/datum/language/dunmeri, //Skyrat change - dunmer language
+		/datum/language/moffic, //Skyrat change - insect language
+		/datum/language/neokanji, //Skyrat change - generally asian language
+		/datum/language/shadowtongue, //Skyrat change - shadowpeople language
+		/datum/language/solcommon, //Skyrat change - sol common blah blah
+		/datum/language/sylvan, //Skyrat change - plantpeople langauge
+		/datum/language/technorussian //Skyrat change - russian sterytype language
```
Comment (CinderWC) [inline code/modules/surgery/organs/tongue.dm:36]: Spell Check (Yes this is the only one i checked, i only care about Russian)

Comment (BobJoga) [reply code/modules/surgery/organs/tongue.dm:36]: fix xed

File: code/controllers/subsystem/language.dm [hunk 2]
```diff
@@ -15,4 +16,28 @@ SUBSYSTEM_DEF(language)
 
 		GLOB.language_datum_instances[language] = instance
 
+		languages_by_name[initial(language.name)] = new language
+
 	return ..()
+
+/datum/controller/subsystem/language/proc/AssignLanguage(mob/living/user, client/cli)
```
Comment (Akrilla) [inline code/controllers/subsystem/language.dm:23]: skyrat changes for this file

**Comment by Akrilla** [comment]
okay there's a few missing skyrat changes, looks alright otherwise

File: code/controllers/subsystem/language.dm (hunk 2 above)
Comment (BobJoga) [reply code/controllers/subsystem/language.dm:23]: will properly annotate it

File: code/modules/language/language_holder.dm [hunk 3]
```diff
@@ -300,21 +304,29 @@ Key procs
...
 
 /datum/language_holder/synthetic
 	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
 								/datum/language/machine = list(LANGUAGE_ATOM),
 								/datum/language/draconic = list(LANGUAGE_ATOM),
 								/datum/language/slime = list(LANGUAGE_ATOM),
+								/datum/language/solcommon = list(LANGUAGE_ATOM),
+								/datum/language/technorussian = list(LANGUAGE_ATOM),
+								/datum/language/neokanji = list(LANGUAGE_ATOM),
+								/datum/language/dunmeri = list(LANGUAGE_ATOM),
```
Comment (jjpark-kb) [inline code/modules/language/language_holder.dm:321]: skyrat change
```suggestion
								// SKYRAT EDIT - additional languages
								/datum/language/solcommon = list(LANGUAGE_ATOM),
								/datum/language/technorussian = list(LANGUAGE_ATOM),
								/datum/language/neokanji = list(LANGUAGE_ATOM),
								/datum/language/dunmeri = list(LANGUAGE_ATOM),
```

File: modular_skyrat/code/modules/mob/living/carbon/human/species_types/ipc.dm [hunk 4]
```diff
@@ -66,4 +67,5 @@
 
 /datum/species/ipc/on_species_loss(mob/living/carbon/human/C)
 	..()
-	C.remove_language(/datum/language/machine)
+	//C.remove_language(/datum/language/machine)
```
Comment (Akrilla) [inline modular_skyrat/code/modules/mob/living/carbon/human/species_types/ipc.dm:70]: we're in modular skyrat we don't need these

File: modular_skyrat/code/modules/mob/living/carbon/human/species_types/synths.dm [hunk 5]
```diff
@@ -95,7 +95,7 @@
 
 /datum/species/synth/on_species_gain(mob/living/carbon/human/H, datum/species/old_species)
 	. = ..()
-	H.grant_language(/datum/language/machine)
+	//H.grant_language(/datum/language/machine) SKYRAT CHANGE= We have an additional language option for this
```
Comment (Akrilla) [inline modular_skyrat/code/modules/mob/living/carbon/human/species_types/synths.dm:98]: can just remove etc, there's a few examples of this

File: modular_skyrat/code/modules/mob/living/carbon/human/species_types/vox.dm [hunk 6]
```diff
@@ -26,10 +26,10 @@
 	H.internal = H.get_item_for_held_index(2)
 	H.update_internals_hud_icon(1)
 	return 0
-
+/*
```
Comment (Akrilla) [inline modular_skyrat/code/modules/mob/living/carbon/human/species_types/vox.dm:29]: here

**Comment by Akrilla** [comment]
will merge after conflict, and above is done

**Pull request merged**